### ISSUE TITLE
Can you give me an example.

### ISSUE THREAD

**Issue by zyh900908** [yohaimagen/gamma_sentinel#1]
Hello,I am a new InSARer,I  find you code,but I cannot understand.Can you give me a example,thsnks

**Comment by yohaimagen** [comment]
Do you have gamma remote sensing software? 
https://www.gamma-rs.ch/

**Comment by zyh900908** [comment]
Yes,I have.I don't know how to use your script to run.Can you give me an example.

**Comment by yohaimagen** [comment]
note this help message you will get when you will run 

python sentinel_possesor.py -h
______________________________
usage: sentinel_possesor.py [-h] [-s S] [-e E] [--m_name M_NAME]
                            [--s_name S_NAME] [--pol POL] [--r_looks R_LOOKS]
                            [--a_looks A_LOOKS] [--cc CC] [--cc_win CC_WIN]
                            [--clean CLEAN] [--orbit_files ORBIT_FILES]
                            master slave

This program process 2-pass sentinel interferogram from raw data to unwraped
phase

positional arguments:
  master                Master sentinel SAR image data base dir path
  slave                 Slave sentinel SAR image data base dir path

optional arguments:
  -h, --help            show this help message and exit
  -s S                  start processing at one of -- Setupprocdir','Import_SL
                        Cs','Download_DEM','Import_DEM','getWidth','DEM_geocod
                        e','Resampling','getDEMwidth','Interferogram','Flatten
                        ing','Filtering','Unwrapping','Geocoding_back','Make_h
                        eaders','Disp
  -e E                  end processing at one of -s following process
  --m_name M_NAME       name of master file, if not spesipy taken to by master
                        dir name
  --s_name S_NAME       name of slave file, if not spesipy taken to by slave
                        dir name
  --pol POL             SLC polarization to extract (hh,hv,vh,vv) default vv
  --r_looks R_LOOKS     number of range looks default: 20
  --a_looks A_LOOKS     number of azimuth looks default: 4
  --cc CC               unwraping coherince threshold default: 0.2
  --cc_win CC_WIN       coherince window size(in pixels) default: 5
  --clean CLEAN         delete all but output files defualt: false
  --orbit_files ORBIT_FILES
                        path to orbit files directory, if not spesfied assume
                        their is a direcory named 'orbit_files' in the working
                        direcory 
_________________________

you should have your working directory organized as so:
-master_file
-slave_file
--orbite_file_dir
----master_orbit_file
----slave_orbit_file

and then run
python sentinel_possesor.py  master_file slave_file

**Comment by zyh900908** [comment]
Dear,I have a problem when I run python sentinel_possesor.py master_file slave_file.
This is where my data is stored.
![image](https://user-images.githubusercontent.com/60143381/73449617-84cad300-439e-11ea-9667-1ebf4f6cec38.png)
and when I run python sentinel_possesor.py master_file slave_file
![image](https://user-images.githubusercontent.com/60143381/73449643-9b712a00-439e-11ea-9c45-55cc03143e66.png)

What should I do.

**Comment by zyh900908** [comment]
Hello, can I see how you type at the terminal.For example,what should I type after python sentinel_possesor.py.(python sentinel_possesor.py ......)
